packages/tools/webpack-fluid-loader/src/routes.ts

```diff
@@ -38,9 +38,9 @@ export function devServerConfig(baseDir: string, env: RouteOptions) {
             static: {
                 directory: path.join(
                     baseDir,
-                    "/node_modules/@fluid-tools/webpack-fluid-loader/dist/fluid-loader.bundle.js",
+                    "/node_modules/@fluid-tools/webpack-fluid-loader/dist/",
                 ),
-                publicPath: "/fluid-loader.bundle.js",
+                publicPath: "/code",
```

Comment:
>"/ [](http://example.com/codeflow?start=28&length=2)

nit: why are we changing this ?

Reply:
What this was doing was just including the fluid loader bundle which it is creating in the comment below in the script in the webpack. However, with our dynamic import we are having more than 1 module. So we are saying these module will be found in the code directory and then below including the fluid loader bundle in the code directory. Our socket bundle will also be found there.